ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
```

Comment:
```suggestion
As the administrator, you are the point of contact if any contributors experience issues with the repository.

In addition, you are in charge of the following:
* Maintaining the Style Guide site on GitHub
* Driving resolution of open issues
* Driving completion of pull requests
```

Reply:
Instead of "In addition, you are in charge of the following:" in @stoobie's suggested change, how about "You also have the following responsibilities:"